app/summary.py

```diff
@@ -0,0 +1,50 @@
+import pickle
+import pandas as pd
+from datetime import datetime
+
+from jjit import search_jjit
+from pracuj import search_pracuj
+
+db_path = 'offers_db'
```

Comment:
To jest stała, więc należy nazwać wielkimi literami, a najlepiej wynieść do pliku konfiguracyjnego

Reply:
Poza tym, takie pickle z danymi powinny dostać oddzielny folder